backend/protos/mgmt/v1alpha1/connection.proto

```diff
@@ -48,6 +48,19 @@ message CheckConnectionConfigRequest {
   ConnectionConfig connection_config = 1;
 }
 
+message CheckConnectionConfigByIdRequest {
+  // The connection ID whose connection we want to validate
```

Comment:
Maybe something simpler like this? the `whose connection` is a little strange since a connection isn't really a person.
```suggestion
  // The connection id to validate
```

Reply:
much better :)